README.md

```diff
@@ -4,7 +4,7 @@ Please see http://godoc.org/github.com/spacemonkeygo/openssl for more info
 
 ### License
 
-Copyright (C) 2017. See AUTHORS.
+Copyright (C) 2020 Northern.tech AS & others. See AUTHORS.
```

Comment:
Should we also add ourselves to the AUTHORS file, maybe with contact@northern.tech ?

Reply:
@kacf I don't really know. Maybe.

But not only AUTHORS, I think we could add a more general note in README explaining that this is a project based on spacemonkeygo and extended by Northern.tech adding functionality for more OpenSSL function calls.

Reply:
:+1: for the general README note

Reply:
:+1:

Reply:
Added. Please review.